src/test/java/org/broadinstitute/hellbender/utils/nio/ExtremeReadsTest.java

```diff
@@ -0,0 +1,106 @@
+package org.broadinstitute.hellbender.utils.nio;
+
+import com.google.common.base.Stopwatch;
+import java.nio.ByteBuffer;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Executor;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+/**
+ * Stress test for reading lots of data from the cloud using a very small prefetch buffer.
+ * Do not run this too often.
+ */
+public final class ExtremeReadsTest extends BaseTest {
+
+    static String fname = GCS_GATK_TEST_RESOURCES + "large/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam";
+
+    static int THREAD_COUNT = 1000;
+    static int CHANNELS_PER_THREAD = 1000;
+    static volatile int errors = 0;
```

Comment:
Make `errors` an instance variable rather than a mutable static variable.

Reply:
OK!

Reply:
Actually on second thought, let's not. If it's not static then Runner cannot access it unless we make Runner non-static or manually pass it a reference to the ExtremeReadsTest instance.